# Практика #1
## Можно ли использовать следующую конструкцию (try без catch и finally):
```
try {
  ...
}
```

Нет, компилятор выдаст ошибку, если использовать блок try без хотя бы одного из этих блоков. блок try предназначен для выполнения кода, который может выбросить исключение. Чтобы обрабатывать это исключение, нужен хотя бы один блок catch, или для гарантированного выполнения кода, который не зависит от того, произошло исключение или нет, нужен блок finally.

# Практика #2
## Можно ли использовать следующую конструкцию (try только с finally):

```
try {
  ...
} finally {
  ...
}

```
Да, она может быть полезна в ситуациях, когда необходимо выполнить код, который должен выполняться всегда (например, освобождение ресурсов), независимо от того, произошло ли исключение.

# Практика #3
## Можно ли использовать следующую конструкцию (try с двумя finally):

```
try {
  ...
} finally {
  ...
} finally {
  ...
}
```

В Java нельзя использовать два блока finally в одном блоке try. Это вызовет ошибку компиляции.